### DISCUSSION TITLE
ValueError Persists.

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1609] [Q&A] [answered]
![Screenshot from 2022-07-27 13-30-13](https://user-images.githubusercontent.com/77268333/181287422-045ce679-5fd3-43bf-8537-7fb9ef71fc3f.png)
![Screenshot from 2022-07-27 13-29-49](https://user-images.githubusercontent.com/77268333/181287459-dafc368b-80ff-4176-a443-4f01832b470e.png)
![Screenshot from 2022-07-27 13-29-14](https://user-images.githubusercontent.com/77268333/181287485-16fbae45-9f77-4c3d-a1f5-d08b3d923e63.png)

The value error message "ValueError: sender doesn't have enough funds to send tx. The upfront cost is: 9435940000000000 and the sender's account only has: 0" Persists even after I change value in get_account().

And also how can I delete or update the local ganache network that is added to the Network List.

**Comment by cromewar** [comment, marked as answer]
Hello @Akhil-franko 
Well could you please be sure to add correctly that network?
Please delete it and add it again, connecting correctly the RPC ganache cliente.
Additionally, take in mind the good practices for python coding, as your functions should be called `get_account` instead of `getAccount`

**Reply by akhilparakka** [reply]
Thank You.
